app/assets/stylesheets/alerts.scss

```diff
@@ -0,0 +1,23 @@
+  //errors
+#error_explanation {
```

Comment:
HTML elements with ID selector `#` should be avoided in most of the cases. What happens if I want to have two error explanations on the same page?

> The id attribute specifies a unique id for an HTML element (the value must be unique within the HTML document).